koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -867,6 +867,12 @@ def handler(self, src, target, opts=None):
         if not SCM.is_scm_url(src):
             raise koji.BuildError('Invalid source specification: %s' % src)
 
+        # don't check build nvr for autorebuild as they might be using
+        # add_timestamp_to_release
+        triggered_after_koji_task = opts.get('triggered_after_koji_task', None)
+        if triggered_after_koji_task:
+            expected_nvr = None
```

Comment:
Although code coverage passes through here, is this being properly tested?

Reply:
added test